packages/help-center/src/components/help-center-support-chat-message.tsx

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable no-restricted-imports */
 import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { Gravatar, TimeSince } from '@automattic/components';
-import { getNumericDateTimeString, useLocale } from '@automattic/i18n-utils';
```

Comment:
This was the only usage of `getNumericDateTimeString` in the entire project. Can we remove it from `i18n-utils` package?